docs/content/en/contribute/releasing/_index.md

```diff
@@ -118,10 +119,14 @@ The diff should look like this:
      version: 0.29.0
 ```
 
-Commit and push the changes:
+Commit:
 ```shell
 git add config/deployments/default/kustomization.yaml charts/k8ssandra-operator/Chart.yaml
 git commit -m"Release v${RELEASED_MAJOR_MINOR?}.0"
+```
+
+Push the changes:
+```shell
```

Comment:
[nitpick] Unrelated, but I like to double-check my commits before I push.
Separate code blocks make it easier to copy-paste the commands.